### ISSUE TITLE
Better timer-emergency management on CLI

### ISSUE THREAD

**Issue by guelzow** [brainworxx/kreXX#32]
Add the possibility to disable the maximum run time check (maxruntime).

Tobi

**Comment by guelzow** [comment]
The idea behind this one was , that when on CLI, there is a much longer execution time.
kreXX starts the time measurement, when it was actually called, and not when the request was started (which may be another issue).
When running on cli for several minutes, this will of cause fail.

The solution is actually quite simple: 
- When on CLI, we reset the timer on every analysis call.
- When on HTML, we reset the timer only once, on the first analysis.

This way, the emergency stop will still work.

Tobi

**Comment by guelzow** [comment]
Oh, will be available with the next version.

Tobi
